### DISCUSSION TITLE
Extra params to mpv and fzf

### DISCUSSION THREAD

**Discussion by guilhermeprokisch** [pystardust/ytfzf#521] [Q&A] [answered]
I want to add extra params to MPV like --autofit=100%  and also to fzf. There's any way to do it without forking?

Thanks for this amazing tool!

**Comment by Euro20179** [comment, marked as answer]
for fzf:
in your ~/.config/ytfzf/conf.sh you can put `FZF_DEFAULT_OPTS="--some --fzf --options"`

There is no equivalent for mpv, the best way to do it would be to add this function to ~/.config/ytfzf/conf.sh

```sh
video_player () {
    [ "$is_detach" -eq 1 ] && use_detach_cmd=detach_cmd || use_detach_cmd=''
    $use_detach_cmd mpv --ytdl-format="$ytdl_pref" --some-extra-mpv-opts "$@"
}
```

This will not work for audio because you'd need to define the `audio_player` function, but I'm assuming you're not using --autofit in audio only.


Writing this has made me realize that I should make `ytfzf` handle the setting of `$use_detach_cmd`, so all you'd have to add is the 2nd line in the function.
